node/validator.go

```diff
@@ -92,10 +121,27 @@ func (n *validator) MevRunning() bool {
 func (n *validator) refresh() {
 	mevRunning, err := n.client.MevRunning(context.Background())
 	if err != nil {
+		metrics.ChainError.Inc()
 		log.Errorw("failed to fetch mev running status", "err", err)
 	}
 
 	n.mevRunning = mevRunning
+
+	balance, err := n.client.BalanceAt(context.Background(), n.account.Address(), nil)
+	if err != nil {
+		metrics.ChainError.Inc()
+		log.Errorw("failed to fetch validator account balance", "err", err)
+	}
+
+	n.account.SetBalance(balance)
```

Comment:
It is possible to have race condition with `GetBalance`?

Reply:
using atomic